Sources/Hub/Hub.swift

```diff
@@ -34,75 +34,6 @@ public extension Hub {
     }
 }
 
-// MARK: - Configuration files with dynamic lookup
-
-@dynamicMemberLookup
```

Comment:
You might be planning this already but `@dynamicMemberLookup` seems to be missing from your new config struct. Would be nice to keep some of the dot syntax for properties that were previously defined like `vocab`, also general syntactic sugar vs string based access pattern - in your original proposal it was included https://github.com/huggingface/swift-transformers/issues/172#issuecomment-2640525635, has your thinking changed to where that is no longer the case?

Reply:
I can certainly revert the change. My initial experience with the codebase was confusing because it was difficult to quickly determine whether a property was statically defined or dynamically accessed. Using explicit subscript notation clarified this distinction and increased my confidence in relying on the type system for accuracy.

Reply:
I'll let @FL33TW00D @pcuenca et al. weigh in, my main concern besides syntax is backwards compatibility if any public properties (that may be accessed in projects that import this one) become inaccessible from this change. 

For example, this was possible via public methods before this PR:
`config.tokenizerData.model.vocab`
and if merged these errors will propagate and require updating
`Value of type 'Config' has no member 'model'`

Perhaps a path forward could be making those objects with defined properties serializable (and probably defining more common properties that were previously dynamic) and leaving the remaining dynamic properties to your subscript pattern, just one idea.

Reply:
If there is no further input on this topic, I will revert to the original behavior.